packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js

```diff
@@ -66,33 +66,42 @@ export const styles = theme => {
   };
 };
 
-class ExpansionPanelSummary extends React.Component {
-  state = {
-    focused: false,
-  };
+function ExpansionPanelSummary(props) {
+  const {
+    children,
+    classes,
+    className,
+    disabled,
+    expanded,
+    expandIcon,
+    IconButtonProps,
+    innerRef,
+    onBlur,
+    onChange,
+    onClick,
+    onFocusVisible,
+    ...other
+  } = props;
+
+  const [focusedState, setFocusedState] = React.useState(false);
 
-  handleFocusVisible = event => {
-    this.setState({
-      focused: true,
-    });
+  const handleFocusVisible = event => {
```

Comment:
Why aren't we using useCallback here?

Reply:
Seems I missed this component when I added memoization to the components I had done the initial conversion of.

Reply:
If we start using `useMemo` and `useCallback` we might as well switch to `PureComponent` and `React.memo`. Otherwise this effort is mostly wasted.

Reply:
Yeah ok. I'm happy to remove useCallback everywhere.